server/accounts_test.go

```diff
@@ -1406,6 +1406,139 @@ func TestAccountRequestReplyTrackLatency(t *testing.T) {
 	}
 }
 
+// This will test for leaks in the remote latency tracking via client.rrTracking
+func TestAccountTrackLatencyRemoteLeaks(t *testing.T) {
+	optsA, _ := ProcessConfigFile("./configs/seed.conf")
+	optsA.NoSigs, optsA.NoLog = true, true
+	srvA := RunServer(optsA)
+	defer srvA.Shutdown()
+	optsB := nextServerOpts(optsA)
+	optsB.Routes = RoutesFromStr(fmt.Sprintf("nats://%s:%d", optsA.Cluster.Host, optsA.Cluster.Port))
+	srvB := RunServer(optsB)
+	defer srvB.Shutdown()
+
+	checkClusterFormed(t, srvA, srvB)
+	srvs := []*Server{srvA, srvB}
+
+	// Now add in the accounts and setup tracking.
+	for _, s := range srvs {
+		s.SetSystemAccount(globalAccountName)
+		fooAcc, _ := s.RegisterAccount("$foo")
+		fooAcc.AddServiceExport("track.service", nil)
+		fooAcc.TrackServiceExport("track.service", "results")
+		barAcc, _ := s.RegisterAccount("$bar")
+		if err := barAcc.AddServiceImport(fooAcc, "req", "track.service"); err != nil {
+			t.Fatalf("Failed to import: %v", err)
+		}
+	}
+
+	// Test with a responder on second server, srvB. but they will not respond.
+	cfoo, crFoo, _ := newClientForServer(srvB)
+	defer cfoo.nc.Close()
+	fooAcc, _ := srvB.LookupAccount("$foo")
+	if err := cfoo.registerWithAccount(fooAcc); err != nil {
+		t.Fatalf("Error registering client with 'foo' account: %v", err)
+	}
+
+	// Set new limits
+	fooAcc.SetAutoExpireTTL(time.Millisecond)
```

Comment:
This seem really low for tests on Travis, no?

Reply:
Not imer related, simply for the prune test.